components/PageView/PageView.spec.tsx

```diff
@@ -4,12 +4,14 @@ import PageView from './PageView';
 
 describe(`PageView`, () => {
   const props = {
-    created: 'test@test.com',
-    title: 'Foo',
-    type: 'Bar',
-    date: '05/06/2020',
-    team: 'Foo 1223',
-    note: 'Foo Bar',
+    formData: {
+      created: 'test@test.com',
```

Comment:
Did you agree with the BEers to call it `created`, because for the other cases we have `created_by`. We should stick with the same structure.

Other possible alignments could be:

`data` => `timestamp` or `date_of_event`

`title` => `case_note_title`

`type` => `form_name`

Reply:
These were only created for the demo as I didn't have access to the rest of the data and assumed it would be different from our current case note data